express-mongoose/src/db/connection.js

```diff
@@ -1,6 +1,6 @@
 const mongoose = require('mongoose');
 
-//mongoose.connect('mongodb://127.0.0.1:27017/Students') // when using natively
-mongoose.connect('mongodb://mongoDb:27017/Students') // When using with docker
+mongoose.connect('mongodb://127.0.0.1:27017/Students') // when using natively
+// mongoose.connect('mongodb://mongoDb:27017/Students') // When using with docker
```

Comment:
Since we have setup our sample-app natively in readme we need to update the mongoDB host should be mongoDb:27017, so people following the readme doesn't face any error.

Reply:
@Sonichigo updated the same